### ISSUE TITLE
Export SocialProviders and add PendingCodeVerification as Interruption

### ISSUE THREAD

**Issue by joleb** [SAP/gigya-react-native-plugin-for-sap-customer-data-cloud#19]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud@0.0.7` for the project I'm working on.

I was trying to setup a user-registration-flow inside an app and came across all the possible [error codes](https://help.sap.com/docs/SAP_CUSTOMER_DATA_CLOUD/8b8d6fffe113457094a17701f63e3d6a/416d41b170b21014bbc5a10ce4041860.html?locale=en-US). In my case i needed the `PendingVerification` one and also all the SocialProviders, to show the supported once inside my app.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/index.d.ts b/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/index.d.ts
index d7cc792..e6da427 100644
--- a/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/index.d.ts
+++ b/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/index.d.ts
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { GigyaDictionary, GigyaInterruption, GigyaSocialProviders } from './Models';
 import { ResolverFactory} from './Resolvers'
 
-export { GigyaInterruption }
+export { GigyaInterruption, GigyaSocialProviders }
 
 export class GigyaError {
     callId: string;
diff --git a/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/index.js b/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/index.js
index d80fe1d..f49c817 100644
--- a/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/index.js
+++ b/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/index.js
@@ -275,7 +275,8 @@ export class GigyaError {
 export const GigyaInterruption = {
     pendingRegistration: 206001,
     pendingVerification: 206002,
-    conflictingAccounts: 403043
+    conflictingAccounts: 403043,
+    pendingCodeVerification: 206006,
 }
 
 export const Gigya = new GigyaInterface();
diff --git a/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/models.ts b/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/models.ts
index 73cceb1..cebcab6 100644
--- a/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/models.ts
+++ b/node_modules/@sap_oss/gigya-react-native-plugin-for-sap-customer-data-cloud/src/models.ts
@@ -7,4 +7,5 @@ export enum GigyaInterruption {
     pendingRegistration = 206001,
     pendingVerification = 206002,
     conflictingAccounts = 403043,
+    pendingCodeVerification = 206006,
 }
\ No newline at end of file
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by tal-mi** [comment]
Fixed in v0.1.0

**Comment by joleb** [comment]
@tal-mi i guess the interruption code for `pendingCodeVerification ` are missing in the v0.1.0 release (as mentioned here: https://help.sap.com/docs/SAP_CUSTOMER_DATA_CLOUD/8b8d6fffe113457094a17701f63e3d6a/929ca94d9a484d7bb466d0746e449a01.html?locale=en-US&q=206006#errors)
